packages/desktop-client/src/components/reports/reports/CustomReport.jsx

```diff
@@ -178,6 +178,24 @@ export function CustomReport() {
 
   const data = { ...graphData, groupedData };
 
+  const items = {
```

Comment:
Could we call this something more informative that items?  Everything else looks fine

Reply:
Good call. Thanks!

It's done.